boards/arm/kinetis/freedom-k64f/README.txt

```diff
@@ -448,8 +448,8 @@ SD Card Support
     Device Drivers -> MMC/SD Driver Support
       CONFIG_MMCSD=y                        : Enable MMC/SD support
       CONFIG_MMSCD_NSLOTS=1                 : One slot per driver instance
-      CONFIG_MMCSD_MULTIBLOCK_DISABLE=y     : (REVISIT)
-      CONFIG_MMCSD_HAVE_CARDDETECT=y         : Supports card-detect PIOs
+      CONFIG_MMCSD_MULTIBLOCK_LIMIT=1       : (REVISIT)
```

Comment:
Is this related to `(REVISIT)` comment?

Reply:
Yes we had issues on the kinetis with CONFIG_MMCSD_MULTIBLOCK_DISABLE=n

Reply:
Yes. I mean is this a "revisit" PR and comment can be updated, or it is still relevant?

Reply:
This is not a "revisit" PR, comment is changed only because of configuration name changed.